src/docs/user.docs.js

```diff
@@ -162,3 +162,26 @@
  *          200:
  *              description: Success
  */
+
+/**
+ * @swagger
+ * /users/disable:
+ *  patch:
+ *    tags:
+ *      - Authentication
+ *    summary: Disables a User
```

Comment:
may you check in your documentation why there is no required parameter `id` representing a user to be disabled 
because using swagger UI it is not working as expected

Reply:
requested changes done